README.md

```diff
@@ -105,6 +105,15 @@ module.exports = (on, config) => {
 };
 ```
 
+## Plugin Options
+
+| Name                | Description                                                                                                                                                                                                                                                | Type                          |
+| ------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------- |
+| `build`             | Sets a build ID. <br> Default: `''`                                                                                                                                                                                                                        | `string`                      |
+| `tags`              | Tags to add to the uploaded Sauce job. <br> Default: `[]`                                                                                                                                                                                                  | `string[]`                    |
+| `region`            | Sets the region. <br> Default: `us-west-1`                                                                                                                                                                                                                 | `us-west-1` \| `eu-central-1` |
+| `artifactUploadDir` | If specified, automatically upload files from this directory, **per spec**. e.g. files in `{artifactUploadDir}/{specName}/` would be uploaded to the job that ran `spec_name`. The directory is relative to your cypress config file. Default: `undefined` | `string`                      |
```

Comment:
Should we mention that `{artifactUploadDir}` is deleted at the beginning of a run?